makefile

```diff
@@ -14,6 +14,7 @@ DEPS_DIR = $(FIDASIM_DIR)/deps
 TABLES_DIR = $(FIDASIM_DIR)/tables
 LIB_DIR = $(FIDASIM_DIR)/lib
 DOCS_DIR = $(FIDASIM_DIR)/docs
+PYTHON_DIR = $(shell which python)$
```

Comment:
Could you change the the variable name to `PYTHON_EXEC` since it is the python executable and not the python directory.

Also I don't think the `$` at the end is necessary.